README.md

```diff
@@ -6,46 +6,217 @@
 
 ![](src/main/resources/META-INF/pluginIcon.svg)
 
-This is a port of [git-machete](https://github.com/VirtusLab/git-machete#git-machete) into an IntelliJ plugin.
+Git Machete plugin is a robust tool that simplifies your git related workflow.
+It's a great complement of IntelliJ version control system.
+You can with ease rebase, pull, push or reset branches.
+Just use discover action or manually define what relation between your branches you want to keep and leave the rest to Git Machete.
+It gives you a simple view of what's going on in your repository.
+Just look at the edge colors or status descriptions to see what should be rebased, pushed, pulled or reset.
 
-This plugin is available on [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/14221-git-machete).
+With this plugin you can simply maintain small pull request that are easy to review.
```

Comment:
`...a maintain small pull requests that...`

Reply:
Hmm I don't think this one is valid, why `a maintain`? :hushed:

Reply:
whoops, most likely I meant `...maintain a small pull request that...` but it should by plural anyway